_config.yml

```diff
@@ -4,7 +4,7 @@
 # Make our URLs look nice. We're not a blog with dated posts, we just want the title.
 permalink:     /:title.html
 
-gems:
+plugins:
```

Comment:
actually, @gunjan5 is right. @emanic - if you run `make htmlproofer` you'll see the broken links.